### PR TITLE
Adds legal statements to the copyright

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2788](https://openedx.atlassian.net/browse/DOC-2788)

Adds the CC-by-SA attribution to the copyright statement in all edX guides.

### Reviewers
- [x] Subject matter expert: Tena Herlihey
- [ ] Subject matter expert: @jbarciauskas 
- [x] Doc team review (sanity check): @catong @srpearce @pdesjardins 

### Testing
- [ ] Ran ./run_tests.sh without warnings or errors

### Other Version
- [x] Google doc with screen shots of proposed changes: https://docs.google.com/a/edx.org/document/d/1FFV3LaufM-sn6aRYZeVdb90m5nQyUakEYT2qxMHgZ-A/edit?usp=sharing

### Post-review
- [ ] Add description to release notes task as a comment
- [x] Squash commits

File: shared/conf.py [hunk 1]
```diff
@@ -41,7 +41,8 @@
 
 # General information about the project.
 
-copyright = u'2016, edX'
+copyright = u'2016, edX Inc. All rights reserved. This work is licensed under a Creative Commons Attribution 4.0 International License'
```
Comment (pdesjardins) [inline shared/conf.py]: I would add a final period here. I see that the wording and capitalization match the Google doc. I think there's a tiny period there also.

**Comment by pdesjardins** [comment]
Looks good. :+1:

File: shared/conf.py (hunk 1 above)
Comment (lamagnifica) [inline shared/conf.py]: Surprisingly, the template adds a period. So, when I first added this new sentence and included a terminal period, it rendered with two periods. Happy to demo if you're interested @pdesjardins as I'd like to understand this better.

**Pull request merged**